src/robot/model/control.py

```diff
@@ -14,10 +14,10 @@
 #  limitations under the License.
 
 from robot.utils import setter
-
 from .body import Body, BodyItem, Branches
 from .keyword import Keywords
 
+from typing import Any, Dict
 
 @Body.register
```

Comment:
- Standard module imports should be before internal imports.
- Our style is to have an empty row between absolute imports from top level module (`from robot.xxx import ...`) and relative imports from the same package (`from .xxx import ...`).
- Two empty lines needed before class declaration (and a class decorator is part of the declaration).